core/runtime/bindings.rs

```diff
@@ -131,6 +131,13 @@ where
 }
 
 pub mod v8_static_strings {
+  pub fn new<'s>(
+    scope: &mut v8::HandleScope<'s>,
+    str_: &'static [u8],
+  ) -> v8::Local<'s, v8::String> {
+    v8::String::new_external_onebyte_static(scope, str_).unwrap()
+  }
+
   pub static DENO: &[u8] = b"Deno";
```

Comment:
Something we should consider (in the future): Aapo added OneByteConst strings that can be const, and do a bit of the v8 string setup at compile time. These strings would be ideal for that.

Reply:
Opened https://github.com/denoland/deno_core/issues/527